package.json

```diff
@@ -22,6 +22,7 @@
   "devDependencies": {
     "bulma": "^0.9.4",
     "es6-promise": "^4.2.8",
+    "npx": "^10.2.2",
```

Comment:
Not needed, it's included with Npm

Reply:
ok. It was not installed when I first tried it.